### ISSUE TITLE
CIT168 node count

### ISSUE THREAD

**Issue by andreifoldes** [PennLINC/xcp_d#942]
## Summary
I realize its rude to ask question about "dev" features, but when opening the `atlas-4S152Parcels_dseg.tsv` I noticed that there are 28 (lateralized) nodes, but in the original CIT168 there are 20 (bilateral) nodes: https://neurovault.org/collections/3145/.

Is this because they get squished in MNI template space?

**Comment by tsalo** [comment]
I believe so. I think @mattcieslak handled the initial resampling, so he could speak to that, but it makes sense that we would lose some regions when downsampling to 2 mm3 and warping to MNI152NLin6Asym.

**Comment by mattcieslak** [comment]
In the AtlasPack repo you can see all the steps that were taken to make these atlases. I don't think, particularly in BOLD, that averaging signal from analogous regions in different hemispheres is what most people intend to do. [This](https://github.com/PennLINC/AtlasPack/blob/main/CIT168/split_up_cit168.py) script is where the splitting and renaming happens.

There are also lines in that script where you can see that some tiny structures are merged together to prevent the downsample-disappearance that can happen when you go to 2 or 3 mm. The names in the sidecars tell you what the original CIT168 regions were and how they're been combined.

**Comment by andreifoldes** [comment]
> In the AtlasPack repo you can see all the steps that were taken to make these atlases. I don't think, particularly in BOLD, that averaging signal from analogous regions in different hemispheres is what most people intend to do. [This](https://github.com/PennLINC/AtlasPack/blob/main/CIT168/split_up_cit168.py?rgh-link-date=2023-09-21T13%3A33%3A57Z) script is where the splitting and renaming happens.

Right! I was trying to understand why there are 14 nodes instead of 20 shown in [neurovault.org/collections/3145](https://neurovault.org/collections/3145/).

> There are also lines in that script where you can see that some tiny structures are merged together to prevent the downsample-disappearance that can happen when you go to 2 or 3 mm. The names in the sidecars tell you what the original CIT168 regions were and how they're been combined.

Yes, thank you for linking that. I can see that node 10 and 11 are merged into node 7, which explains the difference. Would it be possible for XCP_D to consider the resolution of the provided functional files and if sufficient apply the "original" CIT168? If I understand correctly the original atlas was designed with 0.7mm anatomical and was later upsampled to 1mm in template space.

**Comment by tsalo** [comment]
>Would it be possible for XCP_D to consider the resolution of the provided functional files and if sufficient apply the "original" CIT168?

I don't believe the other atlases that compose 4S are available at a higher resolution.
